sdk/timeseriesinsights/Azure.Iot.TimeSeriesInsights/design/Query_API_Design.md

```diff
@@ -0,0 +1,182 @@
+        /// will be executed on the cold store.
+        /// </summary>
+        public StoreType? StoreType { get; set; }
+
+        /// <summary>
+        /// A Time series expression (TSX) filter written as a single string.
+        /// Refer to the documentation on how to write time series expressions.
+        /// </summary>
+        /// <remarks>
+        /// For filter examples, check out the TSX documentation
+        /// <see href="https://docs.microsoft.com/en-us/rest/api/time-series-insights/reference-time-series-expression-syntax">here.</see>.
+        /// </remarks>
+        public string Filter { get; set; }
+
+        /// <summary>
+        /// The maximum number of property values in the whole response set, not the maximum number of property values per page. Defaults to 10,000 when not set. Maximum value of take can be 250,000.
+        /// </summary>
+        public int? MaximumNumberOfEvents { get; set; }
+    }
+
+    /// <summary>
+    /// Optional parameters to use when querying for Time Series Insights events.
+    /// </summary>
+    public class QueryEventsRequestOptions : QueryRequestOptions
+    {
+        /// <summary>
+        /// An array of properties to be returned in the response. These properties must appear
+        /// in the events; otherwise, they are not returned.
+        /// </summary>
+        public EventProperty[] ProjectedProperties { get; set; }
+    }
+
+    /// <summary>
+    /// Optional parameters to use when querying for Time Series Insights series.
+    /// </summary>
+    public class QuerySeriesRequestOptions : QueryRequestOptions
+    {
+        /// <summary>
+        /// Selected variables that needs to be projected in the query result. When it is null or not set, all the
+        /// variables and Time Series types in the model are returned.
+        /// </summary>
+        public string[] ProjectedVariables { get; set; }
+
+        /// <summary>
+        /// Optional inline variables apart from the ones already defined in the Time Series type in the model.
+        /// When the inline variable name is the same name as in the model, the inline variable definition takes precedence.
+        /// </summary>
+        public IDictionary<string, TimeSeriesVariable> InlineVariables { get; set; }
+    }
+    
+    /// <summary>
+    /// Optional parameters to use when querying for Time Series Insights aggregate series.
+    /// </summary>
+    public class QueryAggregateSeriesRequestOptions : QuerySeriesRequestOptions
+    {
+        /// <summary>
+        /// Interval size is specified in ISO-8601 duration format. All intervals are the same size.
+        /// One month is always converted to 30 days, and one year is always 365 days.
+        /// Examples: 1 minute is "PT1M", 1 millisecond is "PT0.001S".
+        /// For more information, <see href="https://www.w3.org/TR/xmlschema-2/#duration"/>
```

Comment:
This will read like:
> For more information, https://www.w3.org/TR/xmlschema-2/#duration

The `<see` does not insert the word "see", so you'll need to add that yourself. Also, punctuation (period) at the end of the sentence.